cypress/integration/participantTable.spec.js

```diff
@@ -0,0 +1,33 @@
+const fixture = require('../fixtures/participantTableTabs.json');
+
+describe('Participant Table', () => {
+  fixture.roles.map((role) => {
+    it(`Correctly renders tabs for ${role.name}`, () => {
+      cy.kcNavAs(role.name, 'participant-view');
```

Comment:
Once Keycloak is working locally, we should be able to replace kcNavAs in place with a regular kcLogin and cy.visit.